app/views/doorkeeper/applications/_delete_form.html.erb

```diff
@@ -1,5 +1,3 @@
-<%- submit_btn_css ||= 'btn btn-link' %>
```

Comment:
This file, and the same type of one in authorized_applications, should be replaced with:

```erb
<%= button_to "Destroy", [:oauth, application],
  :class => "button",
  :data => {
    "abide" => "",
    "confirm" => "Are you sure?"
  },
  :method => :delete %>
```

I think since it's a single element it doesn't need to go in a partial, but that's your call.

The way it currently is set up it uses a normal form tag which has a bottom margin which makes the table's layout all jacked.

This uses the correct Rails helper for the job, and generates a form tag that looks normal.